test/eme.test.js

```diff
@@ -30,6 +30,101 @@ const getMockSession = () => {
 
 QUnit.module('videojs-contrib-eme eme');
 
+QUnit.test('keystatuseschange triggers keystatuschange on player tech for each key', function(assert) {
+  let callCount = {total: 0, 1: {}, 2: {}, 3: {}, 4: {}, 5: {}};
+  const initData = new Uint8Array([1, 2, 3]);
+  const mockSession = getMockSession();
+  const player = {
+    tech_: {
+      trigger: (event) => {
+        if (callCount[event.keyId][event.status] === undefined) {
```

Comment:
Can also just do `if (!callCount[event.keyId][event.status])` or replace the conditional with `callCount[event.keyId][event.status] = callCount[event.keyId][event.status] || 0`

Reply:
Ah yeah, I guess I don't need to worry about `!callCount[event.keyId][event.status]` ever being 0 since I increment it right after.